plugins/c/hu.bme.mit.gamma.xsts.codegeneration.c/src/hu/bme/mit/gamma/xsts/codegeneration/c/HavocBuilder.xtend

```diff
@@ -0,0 +1,208 @@
+/********************************************************************************
+ * Copyright (c) 2018-2023 Contributors to the Gamma project
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ ********************************************************************************/
+package hu.bme.mit.gamma.xsts.codegeneration.c
+
+import hu.bme.mit.gamma.util.GammaEcoreUtil
+import hu.bme.mit.gamma.xsts.codegeneration.c.model.CodeModel
+import hu.bme.mit.gamma.xsts.codegeneration.c.model.HeaderModel
+import hu.bme.mit.gamma.xsts.codegeneration.c.platforms.SupportedPlatforms
+import hu.bme.mit.gamma.xsts.codegeneration.c.serializer.TypeDeclarationSerializer
+import hu.bme.mit.gamma.xsts.model.HavocAction
+import hu.bme.mit.gamma.xsts.model.XSTS
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import org.eclipse.emf.common.util.URI
+
+class HavocBuilder implements IStatechartCode {
+	/**
+	 * The XSTS (Extended Symbolic Transition Systems) used for code generation.
+	 */
+	XSTS xsts;
+	/**
+ 	 * The name of the component.
+ 	 */
+	String name;
+	
+	/**
+	 * The code model for generating code.
+	 */
+	CodeModel code;
+	/**
+	 * The header model for generating code.
+ 	 */
+	HeaderModel header;
+	/**
+	 * Whether our XSTS model contains HavocAction-s or not. Enough to compute once.
+	 */
+	static boolean containsHavocAction = false;
+	
+	/* Serializers used for havoc code generation */
+	val GammaEcoreUtil gammaEcoreUtil = GammaEcoreUtil.INSTANCE
+	val TypeDeclarationSerializer typeDeclarationSerializer = new TypeDeclarationSerializer;
+	
+	/* Boudary definitions */
+	val INT_MIN = 0
+	val INT_MAX = 100
+	
+	val FLOAT_MIN = 0
+	val FLOAT_MAX = 100
```

Comment:
Why are these values used? Why don't we use `INTEGER_MIN` and `INTEGER_MAX`, respectively (and the respective float values where needed)?